package.json

```diff
@@ -43,7 +43,7 @@
     "mocha": "^3.2.0",
     "temp": "^0.8.1",
     "tslint": "^3.15.1",
-    "typescript": "2.2.1"
+    "typescript": "2.3.0"
```

Comment:
I think we should change both.  The peerDependencies are only used from npm, not g3, which is why we didn't notice, but they should be the same value.

(And then run yarn again just to check)